secator/cli.py

```
@@ -1212,11 +1252,12 @@ def report_list(ctx, workspace, runner_type, time_delta, show_all):
 				data['end_date'],
 				data['elapsed'],
 				f"[{status_color}]{data['status']}[/]",
+				_format_vuln_counts(vuln_counts),
 			]
 			if show_all:
 				row.append(str(path))
 			table.add_row(*row)
-		except json.JSONDecodeError as e:
+		except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)